src/generator/AutoRest.Go.Tests/src/tests/generated/body-byte/bytegroupoperations.go

```diff
@@ -0,0 +1,287 @@
+package bytegroup
```

Comment:
>package [](start = 0, length = 7)

Aren't the generated files gitignored? Why do i see them in the last commits or they are different?